display_list/display_list_benchmarks_metal.mm

```diff
@@ -7,16 +7,35 @@
 
 #include "third_party/skia/include/core/SkCanvas.h"
 
+#import <Foundation/NSAutoreleasePool.h>
+
 namespace flutter {
 namespace testing {
 
+class AutoreleasePool {
+ public:
+  AutoreleasePool() : pool_([[NSAutoreleasePool alloc] init]) {}
+
+  ~AutoreleasePool() {
+    [pool_ release];
+    pool_ = nullptr;
+  }
+  void drain() {
+    [pool_ drain];
+    pool_ = [[NSAutoreleasePool alloc] init];
+  }
```

Comment:
This is kind of a weird and you aren't using it.

Reply:
Oh yeah, I took this class directly from Skia's implementation at https://source.chromium.org/chromium/chromium/src/+/main:third_party/skia/tools/AutoreleasePool.mm;l=9?q=autoreleasepool&sq=&ss=chromium%2Fchromium%2Fsrc:third_party%2Fskia%2F